ui/src/message_popup/author/questions_page.jsx

```diff
@@ -59,26 +70,54 @@ export default React.createClass({
     return questionString.split(text).length;
   },
   
+  getSearchResults(){
+    const {searchText} = this.state;
+    if(searchText === '') return this.state.allDatabaseQuestions;
+    const currentQuestions = this.state.allDatabaseQuestions.currentQuestions
+      .filter(question => this.searchQuestionForText(searchText, question))
+      .sort(question => this.countTextOccurrences(searchText, question));
+    const archivedQuestions = this.state.allDatabaseQuestions.archivedQuestions
+      .filter(question => this.searchQuestionForText(searchText, question))
+      .sort(question => this.countTextOccurrences(searchText, question));
+    return {currentQuestions, archivedQuestions};
```

Comment:
This looks good!  I'd suggest factoring the calls to filter/sort out into a method, since it's the same for both sets of questions.